ProcessesApi/V1/Factories/ISnsFactory.cs

```diff
@@ -11,5 +13,6 @@ public interface ISnsFactory
         EntityEventSns ProcessClosed(Process process, Token token, string description);
 
         EntityEventSns ProcessUpdated(Process process, Token token, string description);
+        EntityEventSns ProcessByIdUpdated(Guid id, UpdateEntityResult<ProcessState> updateResult, Token token);
```

Comment:
Maybe rename `ProcessByIdUpdated` to `ProcessUpdated` & `ProcessUpdated` to `ProcessUpdatedWithMessage` or something? To be more clear